tool/tsh/dbcmd.go

```diff
@@ -315,3 +320,18 @@ func (c *cliCommandBuilder) getMongoCommand() *exec.Cmd {
 	// fall back to `mongo` if `mongosh` isn't found
 	return exec.Command(mongoBin, args...)
 }
+
+// getRedisCommand returns redis-cli commands used by 'tsh db connect' when connection to a Redis instance.
+func (c *cliCommandBuilder) getRedisCommand() *exec.Cmd {
+	// TODO(jakub): Add "-3" when Teleport adds support for Redis RESP3 protocol.
+	args := []string{
+		"--tls",
+		"-h", c.options.localProxyHost,
+		"-p", strconv.Itoa(c.options.localProxyPort),
+		"--cacert", c.profile.CACertPathForCluster(c.rootCluster),
+		"--key", c.profile.KeyPath(),
+		"--cert", c.profile.DatabaseCertPathForCluster(c.tc.SiteName, c.db.ServiceName),
+	}
+
+	return exec.Command(redisBin, args...)
```

Comment:
nit: not related to this line but something on tsh `formatConnectCommand`

https://github.com/gravitational/teleport/blob/fbd5a2aafd42389d1cb55a6a98d83a87a6857a7d/tool/tsh/tsh.go#L1548-L1565

we may not need database name if connecting to redis

Reply:
Let me think about it, but I think we can use `--db-name` to connect user to a different db. I could take values 0-15 as that's the only DBs supported by Redis.

Reply:
it can be cool we can select redis database. in that case, maybe `--db-index` or `--db-number` is a better name. if I recall, redis cluster does not support this (like cluster has only 0)?

Reply:
My 2c is I don't think we need an extra flag but being able to say `--db-name=2` (if that's what Redis uses to identify databases) might be useful.

Reply:
I added a client side options to specify db number when using `tsh db connect`.